src/main/java/io/github/thebusybiscuit/slimefun4/core/networks/energy/EnergyNet.java

```diff
@@ -255,8 +255,15 @@ private int tickAllGenerators(@Nonnull LongConsumer timings) {
                     BlockStorage.clearBlockInfo(loc);
 
                     Slimefun.runSync(() -> {
+                        io.github.bakedlibs.dough.config.Config cfg = Slimefun.getCfg();
```

Comment:
This hurts, idk if we can do anything about it, but it hurts

Reply:
If it’s changed to item.yml it’s gone